pkgs/tools/misc/shopware-cli/default.nix

```diff
@@ -0,0 +1,51 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, installShellFiles
+, makeWrapper
+, dart-sass-embedded
+}:
+
+buildGoModule rec {
+  pname = "shopware-cli";
+  version = "0.1.62";
+  src = fetchFromGitHub {
+    repo = "shopware-cli";
+    owner = "FriendsOfShopware";
+    rev = version;
+    hash = "sha256-Vxg3hYUCN4UdBGHN57VXDlGdp+qzq8tbUDPAl/Bse44=";
+  };
+
+  nativeBuildInputs = [ installShellFiles makeWrapper ];
+
+  vendorSha256 = "sha256-O3dJX2w+J9TABkytaJ0ubFg9edVpmo7iPArc+Qqh3M4=";
+
+  postInstall = ''
+    export HOME="$(mktemp -d)"
+    installShellCompletion --cmd shopware-cli \
+      --bash <($out/bin/shopware-cli completion bash) \
+      --zsh <($out/bin/shopware-cli completion zsh) \
+      --fish <($out/bin/shopware-cli completion fish)
+  '';
+
+  preFixup = ''
+    wrapProgram $out/bin/shopware-cli \
+      --prefix PATH : ${lib.makeBinPath [ dart-sass-embedded ]}
+  '';
+
+  CGO_ENABLED = 0;
```

Comment:
Any particular reason why you chose to disable `CGO_ENABLED`? :)

In nixpkgs, CGO is usually kept enabled.
See https://github.com/NixOS/nixpkgs/pull/224819#issuecomment-1500890578

Reply:
Ahh nice to know. I have removed it :)